src/file_management/core.clj

```diff
@@ -1,10 +1,23 @@
 (ns file-management.core
+  (use clojure.java.io)
+  (:import (org.apache.commons.codec.binary Base64))
   (:require [aws.sdk.s3 :as s3]))
 
 (defn build-avatar-key
   [user-id]
   (str user-id "-avatar"))
 
+(defn get-avatar-url
+  [bucket user-id]
+  (str "https://" bucket ".s3.amazonaws.com/" (build-avatar-key user-id)))
+
+(defn get-img-stream
+  [src]
+  (let [src-parsed (clojure.string/replace src #"data:image/png;base64," "")
```

Comment:
This replace can be troublesome on a huge file, maybe `#"^data:image/png;base64,"` (the ^ forces the fixed start search)